src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/CodeAction.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerFx.LanguageServerProtocol.Protocol
+{
+    public class CodeAction
+    {
+        public CodeAction()
+        {
+            Diagnostics = new Diagnostic[] { };
+        }
+
+        public string Title { get; set; }
+
+        public string Kind { get; set; }
+
+        /// <summary>
```

Comment:
ideally add descriptive comments for all these members

Reply:
or links to the LSP docs

Reply:
yes simplified.